packages/magento-cart-coupon/ApplyCouponForm/ApplyCouponForm.tsx

```diff
@@ -54,7 +54,7 @@ export function ApplyCouponForm(props: ApplyCouponFormProps) {
         </Button>
       </FormControl>
 
-      <ApolloCartErrorAlert error={error} />
+      <ApolloCartErrorAlert error={error} resetCart={false} />
```

Comment:
Can you revert this, we've got this covered in another PR and will solve this whole class of errors. The user will never see the 'Reset Cart' button anymore and they wont have a broken cart anymore.